service/dap/server_test.go

```diff
@@ -2377,44 +2377,37 @@ func TestLaunchRequestDefaults(t *testing.T) {
 
 func TestLaunchRequestDefaultsNoDebug(t *testing.T) {
 	runTest(t, "increment", func(client *daptest.Client, fixture protest.Fixture) {
-		runNoDebugDebugSession(t, client, "launch", func() {
+		runNoDebugDebugSession(t, client, func() {
 			client.LaunchRequestWithArgs(map[string]interface{}{
 				"noDebug": true,
```

Comment:
Let's add some bad testing to TestBadLaunchRequests.

Reply:
WDYT